portfolio/src/main/java/com/google/sps/data/Information.java

```diff
@@ -0,0 +1,85 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Information {
```

Comment:
Maybe a better class name would be PostInformation or RecommendationInfo?

Reply:
Honestly I would say "Post" or "Recommendation" would work too. "Info" doesn't really add much, since technically everything we work with is information.

Reply:
Let me change it, How about PostInformation, I think this name from Matthew is OK.